backends/platform/ios7/ios7_osys_main.cpp

```diff
@@ -162,6 +162,11 @@ bool OSystem_iOS7::hasFeature(Feature f) {
 	case kFeatureTouchscreen:
 		return true;
 
+#if defined(__ARM_NEON__) || defined(__aarch64__)
```

Comment:
The Armv7 and Armv7s have NEON support as well, but they are not 64 bit arcs so perhaps the defined(__aarch64__) should be removed